.github/workflows/tox.yml

```diff
@@ -29,4 +29,8 @@ jobs:
         python -m pip install --upgrade pip
         pip install tox tox-gh-actions
     - name: Test with tox
-      run: tox
+      run: tox && cat ./coverage.xml
+    - name: Upload to coverage
+      uses: codecov/codecov-action@v1.0.10
+      with:
+        flags: ${{ matrix.python-version }}
```

Comment:
Maybe try some options:

```suggestion
        flags: ${{ matrix.python-version }}
        file: ./coverage.xml
        fail_ci_if_error: true
```
?

Reply:
I think rather that since we step into riotctrl, `codecov` is unable to construct the report from the coverage.xml. Trying https://docs.codecov.io/docs/fixing-paths first

Reply:
The coverage.xml is found, so I don't think adding the `./coverage.xml` will change anything. I tried that already.

Reply:
And since uploads sometimes seem to fail I also don't want to activate `fail_ci_if_error` and I don't see how it would help with the problem at hand :-/

Reply:
Tried it, but [did not work either](https://codecov.io/gh/RIOT-OS/riotctrl/commit/56c64cf6e528d460af5c36e19da41a5f4026c190)